content/api/entity-reference/gift/index.html

```diff
@@ -123,12 +123,12 @@
             <tr>
                 <td>gift_aid_amount</td>
                 <td><a href="{{ stache.config.constituent_entity_reference }}#Currency">currency</a></td>
-                <td>The amount of tax reclaimed from Gift Aid for this gift. For UK only.</td>
+                <td>The amount of tax reclaimed from Gift Aid for this gift.  For UK formatted databases only. This is a computed field based on total qualified gift aid amount across all <code>gift_splits</code>.</td>
```

Comment:
Why do we only update two of the four Gift Aid entries in the gift entity reference. Shouldn't the entries in the gift table have changes similar to the changes in the gift splits table?

Also, in the constituent entity reference, we just say "For the UK only." We don't have the "formatted databases" in there. I think we should be consistent. My 2 cents is that we should stick with "For the UK only" (including the missing "the"). But if we want to add "formatted databases," I'd do that in all instances, not just this one. And I'd hyphenate "UK-formatted."

I'd also rework the last sentence to avoid the passive "based on" formation. I'd change it to: "This computed field calculates the total qualified gift aid amount across all <code>gift_splits</code>."